plugins/aggregators/basicstats/basicstats_test.go

```diff
@@ -357,3 +434,26 @@ func TestBasicStatsWithUnknownStat(t *testing.T) {
 
 	acc.AssertDoesNotContainMeasurement(t, "m1")
 }
+
+// Test that if Stats isn't supplied, then we only do count, min, max, mean,
+// stdev, and s2.  We purposely exclude sum for backwards compatability,
+// otherwise user's working systems will suddenly (and surprisingly) start
+// capturing sum without their input.
+func TestBasicStatsWithDefaultStats(t *testing.T) {
```

Comment:
This might be overkill, but I like the idea of verifying "default configuration" separately from verifying "aggregations are working".